pyproject.toml

```diff
@@ -0,0 +1,11 @@
+[build-system]
+requires = ["flit_core >=2,<3", "requests = 2.23.0"]
+build-backend = "flit_core.buildapi"
+
+[tool.flit.metadata]
+module = "msgraphcore"
+author = "Microsoft"
+author-email = "graphtooling@microsoft.com"
+classifiers = ["License :: OSI Approved :: MIT License"]
+description-file = "README.md"
```

Comment:
home-page is required. https://flit.readthedocs.io/en/latest/flit_ini.html.

Reply:
Tracked in #35